Source/Data/BPMFMappings.txt

```diff
@@ -45667,6 +45668,7 @@
 巧妙地 ㄑㄧㄠˇ ㄇㄧㄠˋ ㄉㄜ˙
 巧婦 ㄑㄧㄠˇ ㄈㄨˋ
 巧婦難為 ㄑㄧㄠˇ ㄈㄨˋ ㄋㄢˊ ㄨㄟˊ
+巧婦難為無米之炊 ㄑㄧㄠˇ ㄈㄨˋ ㄋㄢˊ ㄨㄟˊ ㄨˊ ㄇㄧˇ ㄓ ㄔㄨㄟ
```

Comment:
I remember the max length of a phrase is 6. Does it work?

Reply:
We do have long phrases like `偷雞不著蝕把米` and `巴布亞紐幾內亞`, but since we have these already, let's remove it.

```
BPMFMappings.txt
45686:巧婦難為 ㄑㄧㄠˇ ㄈㄨˋ ㄋㄢˊ ㄨㄟˊ

BPMFMappings.txt
83538:無米之炊 ㄨˊ ㄇㄧˇ ㄓ ㄔㄨㄟ
```

Reply:
I tested this, and as mentioned, it seems that words longer than 6 characters are not working.
For such long words, we might need to split them in the future into something like:
```
偷雞不著
蝕把米
巴布亞紐
幾內亞
```
https://github.com/user-attachments/assets/b4be21ad-6320-4f03-b153-c7056a965c79

Reply:
These two should be the only phrases with 7+ characters. 

```
$ grep -E '^\S\S\S\S\S\S\S' BPMFMappings.txt
偷雞不著蝕把米 ㄊㄡ ㄐㄧ ㄅㄨˋ ㄓㄠˊ ㄕˊ ㄅㄚˇ ㄇㄧˇ
偷雞不著蝕把米 ㄊㄡ ㄐㄧ ㄅㄨˋ ㄓㄨㄛˊ ㄕˊ ㄅㄚˇ ㄇㄧˇ
巴布亞新幾內亞 ㄅㄚ ㄅㄨˋ ㄧㄚˇ ㄒㄧㄣ ㄐㄧˇ ㄋㄟˋ ㄧㄚˇ
巴布亞新幾內亞 ㄅㄚ ㄅㄨˋ ㄧㄚˇ ㄒㄧㄣ ㄐㄧˇ ㄋㄟˋ ㄧㄚˋ
巴布亞紐幾內亞 ㄅㄚ ㄅㄨˋ ㄧㄚˇ ㄋㄧㄡˇ ㄐㄧˇ ㄋㄟˋ ㄧㄚˇ
巴布亞紐幾內亞 ㄅㄚ ㄅㄨˋ ㄧㄚˇ ㄋㄧㄡˇ ㄐㄧˇ ㄋㄟˋ ㄧㄚˋ
```

Reply:
@xatier please take a look #568